README.md

```
@@ -83,6 +84,50 @@ npm install --legacy-peer-deps
 npm install some-package --legacy-peer-deps
 ```
 
+## AI Prompt for Integration
+
+Paste this prompt into an AI coding assistant (Claude Code, Cursor, etc.) working in the target
+app's repo to wire up `@xola/ui-kit` end to end.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)